### PR TITLE
Feature: handle quick contact form with formspree.io - index.html, ar…

### LINKED ISSUE #129: [BUG] Quick contact form returns 405 error code
* When submitting the quick contact form it shows a 405 HTTP  error code.

#

image

### Expected behaviour

I expected it to display a prompt successful, or return status code 200

### Actual behaviour

Returns `Error: 404 https://osc-ksa.com/forms/contact.php`

### How to reproduce

Dell lattitude 5510 (Windows 11)
VSCODE v1.72
Chrome Dev v108

- OS: Windows 11

### PR DESCRIPTION
**Pull request by RealRichi3**
### Summary
Quick contact form for homepage returns error 405 - issue #129 
Email handler for quick contact form exposes sensitive private keys in front end - issue #130  

For this fix, an account should be created on formspree.io to get the form_id to the passed in the form action url (line 522 for index.html, line 481 for ar-index.html)

Fixes #129 #130  #146

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### List of changes proposed in this PR (pull-request)
* *Added formspree.io as email sending service for quick contact form.*

### What should a reviewer concentrate their feedback on?

- [x] Everything looks ok?

### CODE REVIEW

**Comment by RealRichi3** [comment]
@BatoolMM Please review

File: ar-index.html [hunk 1]
```diff
@@ -478,7 +478,8 @@ <h3 class="section-title">تواصل معنا</h3>
 
           <div class="col-lg-5 col-md-8">
             <div class="form">
-              <form action="forms/contact.php" method="post" role="form" class="php-email-form">
+              <!-- Replace `form_id` with id generated in formspree.io account -->
+              <form action="https://formspree.io/f/{form_id}"method="post" role="form" class="php-email-form">
```
Comment (BatoolMM) [inline ar-index.html:482]:
```suggestion
              <form action="https://formspree.io/f/mqkjddaj"method="post" role="form" class="php-email-form">

```

File: index.html [hunk 2]
```diff
@@ -518,7 +518,8 @@ <h3 class="section-title">Contact Us</h3>
 
           <div class="col-lg-5 col-md-8">
             <div class="form">
-              <form action="forms/contact.php" method="post" role="form" class="php-email-form">
+              <!-- Replace `form_id` with id generated in formspree.io account -->
+              <form action="https://formspree.io/f/{form_id}" method="post" role="form" class="php-email-form">
```
Comment (BatoolMM) [inline index.html:522]:
```suggestion
              <form action="https://formspree.io/f/mqkjddaj" method="post" role="form" class="php-email-form">
```

**Review by BatoolMM — APPROVED** [review]
Thank you @RealRichi3, just added the form's endpoint!

**Comment by BatoolMM** [comment]
I updated the form's endpoint but it seems to still gives the same error, can you review it @RealRichi3 and @alswajiab!

**Comment by BatoolMM** [comment]
Thank you, it's working (email is delivered) but still shows an error in the website, not sure why!

I will merge this for now as quick fix!

**Pull request merged**